templates/home/index.html

```diff
@@ -13,6 +13,8 @@ <h1>{{ applicationName }}</h1>
                 <li>Upload samples that are broken</li>
                 <li>Get information on previously submitted samples</li>
                 <li>Run the regression tests</li>
+                <li><a href="{{ url_for('custom.index')}}"> Run your own tests </a>if you have permissions</li>
```

Comment:
This is a good idea to add it over here, but I'd use an if/else to display one or the other depending on the role.